src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -321,6 +321,271 @@ public async Task TestRemoteWorkspaceCircularReferences()
             Assert.NotNull(solution);
         }
 
+        [Fact]
```

Comment:
i'm looking to see how i can reduce the amount of duplication in all these tests.

all of them share mostly the same structure:  generate something, then tweak teh generator, then show that that doesn't matter since local and oop sync'ed what they did.